packages/origin/src/blockchain-facade/Certificate.ts

```diff
@@ -371,3 +374,29 @@ export class Entity extends TradableEntity.Entity implements ICertificate {
         return returnEvents;
     }
 }
+
+export async function claimCertificates(
+    certificateIds: string[],
+    configuration: Configuration
+) {
+    const certificateIdsAsNumber = certificateIds.map(c => parseInt(c, 10));
+
+    const accountProperties = {
+        from: configuration.blockchainProperties.activeUser.address,
+        privateKey: configuration.blockchainProperties.activeUser.privateKey
+    };
+
+    return configuration.blockchainProperties.certificateLogicInstance.claimCertificateBulk(certificateIdsAsNumber, accountProperties);
+};
+
+export async function claim(
```

Comment:
Would be better to make this a part of `Certificate.Entity`. Wouldn't have to pass the ID every time then, but you could just call `await certificate.claim()`.